notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
<p>If this is multi-class classification with 3 classes, will negating the probabilities give us accurate results?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>I think if we remove the NaN, we will have only two classes. This should solve this problem.</p>

Reply:
<p><code>probs_test = calibrated.predict_proba(testx)[:, 1]</code>  </p><p> </p><p>I assume that this is outputting an array with 3 values, on for each class, why not just collect <code>[:,0]</code>   which is probably the failure class from the onset? </p>

Reply:
<p>Also, are these true probabilities? Like does <code>sum(calibrated.predict_proba(testx)[:, 1])</code>  always equal 1.0?</p>

Reply:
<p>That's a great suggestion. We are on it :)</p>